paddle/fluid/pybind/pybind.cc

```diff
@@ -1339,6 +1339,14 @@ All parameter, weight, gradient are variables in Paddle.
           "num_trainers",
           [](const BuildStrategy &self) { return self.num_trainers_; },
           [](BuildStrategy &self, int num_trainers) {
+#ifdef WIN32
+            if (std::abs(num_trainers) > 1) {
+              LOG_FIRST_N(WARNING, 1)
+                  << "Windows has no support to distribute mode, "
+                     "num_trainers would be forced to 1.";
+              num_trainers = 1;
```

Comment:
Now that Windows doesn't support distribution training, the program should be exited.

Reply:
ok